.github/workflows/convert-to-json.yml

```diff
@@ -0,0 +1,41 @@
+name: Convert to JSON
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  convert-to-json:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip jq yq
+
+    - name: Convert frequency-plans.yml to json
+      run: |
+        yq . frequency-plans.yml >> frequency-plans-generated.json
```

Comment:
1. This can just be on one line (after `run:`)
2. I don't think we should append to the JSON, so use `>` instead of `>>`.